### ISSUE TITLE
Accessibility:File Uploader: Visible text on the button “Drag a .jpg file on me!” is not read, when the button is accessed. 

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3498]
Description: Visible text on the button “Drag a .jpg file on me!” is not read, when the button is accessed. 

Expected: complete name of the UI or text on the button should be read out when accessed. 

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/94651819-ac4fb200-0316-11eb-9d91-79c12626f913.png)

**Comment by puru-hk** [comment]
As this issue raised by accessibility team. Accessibility team has to verify and close this issues. Hence reopening this issue.

**Comment by Sanjay-1312** [comment]
Hello Colleagues,
The issue is fixed.
Hence closing the ticket.
JAWS 2019

Thanks and Regards
Sanjay
